src/pyasm/search/database_impl.py

```diff
@@ -450,7 +450,37 @@ def get_child_cte(my, op_filters):
 
         return stmt
 
- 
+    def get_child_code_cte(my, var_dict):
```

Comment:
it should take in the argument as collection_type, search_type, parent_collection_code.. and you can buidl the var_dict in this method